core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -359,4 +359,5 @@ NoOperator=No operator for ''{0}'' with kind: ''{1}'', syntax: ''{2}'' during JS
 MustFilterFieldsMissing=SQL statement did not contain filters on the following fields: {0}
 IllegalNegativeBitGetPosition=BIT_GET/GETBIT error: negative position {0,number} not allowed
 IllegalBitGetPositionExceedsLimit=BIT_GET/GETBIT error: position {0,number} exceeds the bit upper limit {1,number}
+HavingMayNotContainWindow=Window expressions are not allowed in HAVING
```

Comment:
I think the exception name "HavingMayNotContainWindow" is not very clear. Maybe we can use "WindowInHavingNotAllowed"